uber/config.py

```diff
@@ -1063,7 +1063,11 @@ def _is_intstr(s):
 c.DONATION_TIER_ITEMS = {}
 c.DONATION_TIER_DESCRIPTIONS = _config.get('donation_tier_descriptions', {})
 for _ident, _tier in c.DONATION_TIER_DESCRIPTIONS.items():
-    [price] = [amt for amt, name in c.DONATION_TIERS.items() if name == _tier['name']]
+    try:
```

Comment:
This is to handle cases where [donation_tier_descriptions] including tiers not set in [[donation_tier]].